packages/@ember/-internals/runtime/tests/legacy_1x/mixins/observable/observable_test.js

```diff
@@ -399,13 +397,13 @@ moduleFor(
         assert.equal(object.get(key), key, `Try #2: object.get(${key}) should run function`);
       });
 
-      // verify each call count.  cached should only be called once
+      // verify each call count. cached should only be called once
       w('computedCalls dependentFrontCalls dependentCalls').forEach((key) => {
-        assert.equal(object[key].length, 2, `non-cached property ${key} should be called 2x`);
+        assert.equal(object[key].length, 1, `non-cached property ${key} should be called 1x`);
       });
 
       w('computedCachedCalls dependentCachedCalls').forEach((key) => {
-        assert.equal(object[key].length, 1, `non-cached property ${key} should be called 1x`);
+        assert.equal(object[key].length, 1, `cached property ${key} should be called 1x`);
```

Comment:
This test can be dramatically simplified. The volatile CPs, after the removal of the volatility, are simply plain CPs with cached semantics. There is not need to test those same semantics twice.

Similarly I think `dependentFrontCalls` and `dependentCalls` were at one time differentiated by the argument order to `computed(`. Now there is only one order, so there is no need to assert the same thing twice.

Just take a look at the CPs after your patch. I'm pretty sure they now repeat themselves a few times, just with different inherited-from-old-code variable names.

Reply:
Sounds good. I left it as is because I wasn’t sure what the difference between some of the test CPs were so erred on the side of caution

Reply:
Ok, done (and I think I removed the correct ones 🤞)